pilgrimor/cli/base_cli.py

```diff
@@ -39,11 +42,39 @@ def __call__(self) -> None:
                         f"Can't find {self.namespace.command}.",
                     ),
                 )
-
+            if self.namespace.version:
+                project_version = self._get_project_version()
+                if self.namespace.version != project_version:
+                    exit(
+                        error_text(
+                            f"Migration version: {self.namespace.version} "
+                            f"does not match the project version "
+                            f"in pyproject.toml: {project_version}.",
+                        ),
+                    )
             getattr(self, self.namespace.command)()
         else:
             exit(
                 attention_text(
                     "You don't call any command",
                 ),
             )
+
+    def _get_project_version(self) -> str:
```

Comment:
Let's find out what version in `pyproject.toml` in Settings because in that class we already have instance of parsed  `pyproject.toml` file.

Reply:
reactor via settings